app/src/main/java/com/devshish/internship/presentation/service/player/MediaSessionCallback.kt

```diff
@@ -29,6 +39,42 @@ class MediaSessionCallback(
         playWhenReady = true
     }
 
+    private fun NotificationCompat.Builder.createNotification(): NotificationCompat.Builder {
+        return apply {
+            val controller = mediaSession.controller
+
+            setContentIntent(controller.sessionActivity)
+
+            setDeleteIntent(
+                MediaButtonReceiver.buildMediaButtonPendingIntent(
+                    context,
+                    PlaybackStateCompat.ACTION_STOP
+                )
+            )
+
+            setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
+
+            setSmallIcon(R.drawable.ic_music)
+            color = ContextCompat.getColor(context, R.color.black)
+
+            setSilent(true)
+
+            setStyle(
+                androidx.media.app.NotificationCompat.MediaStyle()
+                    .setMediaSession(mediaSession.sessionToken)
+                    .setShowActionsInCompactView(0)
+                    .setShowCancelButton(true)
+                    .setCancelButtonIntent(
+                        MediaButtonReceiver.buildMediaButtonPendingIntent(
+                            context,
+                            PlaybackStateCompat.ACTION_STOP
+                        )
+                    )
+
+            )
+        }
```

Comment:
And this))

Actually - whole this ext function could be created when you create builder